tensorflow_addons/layers/python/maxout_test.py

```diff
@@ -19,27 +19,27 @@
 from __future__ import print_function
 
 import numpy as np
+import tensorflow as tf
 
-from tensorflow.python.keras import testing_utils
-from tensorflow.python.platform import test
+from tensorflow.python.keras import testing_utils as keras_test_util
```

Comment:
Are the APIs you need not public here? We should probably make them public if they are useful.

Reply:
I don't believe they are. I can add this to the list of APIs that we find we need.